app/javascript/packages/components/status-page.tsx

```diff
@@ -86,7 +86,13 @@ function StatusPage({
 
   return (
     <>
-      <img src={src} width={54} height={54} alt={alt} className="display-block margin-bottom-4" />
+      <img
+        src={src}
+        alt={alt}
+        width={88}
+        height={88}
+        className="display-block margin-bottom-4 alert-icon"
```

Comment:
Would be good to follow-up to align the componentization of AlertIcon in React to match the corresponding ViewComponent implementation.